front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx

```diff
@@ -110,7 +124,15 @@ function CreateShipmentView() {
   const currentOrganisationName = currentBase?.organisation?.name;
   const currentOrganisationBase = currentBase?.name;
   const currentOrganisationId = globalPreferences.organisation?.id;
-  const currentOrganisationBases = globalPreferences.availableBases;
+
+  const AllBasesOfCurrentOrg = useQuery<AllAvailableBasesQuery>(ALL_BASES_OF_CURRENT_ORG_QUERY, {
+    variables: {
+      orgId: currentOrganisationId,
```

Comment:
in the functional test an error toast appears on a reload since the currentOrganisationId is not set while useLoadAndSetGlobalPreferences is still loading.

is there a way quick way to only execute the query one useSetAndLoadGlobalPreferences is done?